tests/core/test_metadata.py

```diff
@@ -39,6 +40,20 @@ def test_saving_with_project(model, tmpdir):
     load_meta(path)
 
 
+def test_saving_with_pathlib(model, tmpdir):
```

Comment:
Unrelated to this PR, but use of `tmpdir` and `tmpdir_factory` is discouraged and `tmp_path`/`tmp_path_factory` is preferred which returns `pathlib.Path` objects.

Reply:
yup, as I mentioned, it's unrelated to your PR and I am not asking for a change. Just pointing out what I noticed.

Reply:
Yeah, thanks. I will CC @aguschin and @mike0sv here to take note as well. Will probably open another issue to take note.